src/layouts/hass-error-screen.ts

```diff
@@ -68,6 +68,7 @@ class HassErrorScreen extends LitElement {
           padding: 8px 12px;
           pointer-events: none;
           background-color: var(--app-header-background-color);
+          -webkit-backdrop-filter: var(--header-backdrop-filter, unset);
```

Comment:
The `app` prefix is used for the background. Should it be added to this variable too?

Reply:
My idea is to make this available for all devices, not app specific

Reply:
I think app here refers to "web app" and not "mobile app".

Reply:
Oh, sounds good then, I've updated